client/hotrod-client/pom.xml

```diff
@@ -25,6 +25,7 @@
       <server.keystore>${project.build.testOutputDirectory}/keystore_server.p12</server.keystore>
       <default.password>secret</default.password>
       <alt.password>changeme</alt.password>
+      <infinispan.test.parallel.threads>5</infinispan.test.parallel.threads>
```

Comment:
Might as well keep as 8 (parent value) since the other changes reduced the memory substantially